unified-codes/apps/web/src/components/templates/Header.tsx

```diff
@@ -63,9 +63,9 @@ const _Header: Header = ({ classes }) => {
       </Grid>
       <Grid item style={{ flex: 1 }}>
         <Grid container className={classes.menu}>
-          <Grid item>
+        <Grid item>
             <Link component={RouterLink} to="/">
-              Browse
+              Search
```

Comment:
I think I'll leave it as it was previously (`Browse`) since it's not really in the context of this issue anymore (originally tried to link directly to the details page from the header and realised it didn't make sense).